app/assets/src/components/views/projects/ProjectsView.jsx

```diff
@@ -67,7 +67,7 @@ class ProjectsView extends React.Component {
         <div className={cs.projectDescription}>{project.description}</div>
         <div className={cs.projectDetails}>
           <span className={cs.projectCreationDate}>
-            {moment(project.created_at).fromNow()}
+            {moment(project.updated_at).fromNow()}
```

Comment:
Indeed `updated_at` seems closer to what @jenn initially intended.
But, for `updated_at` to be changed on new samples, we need to add `touch: true` to `belongs_to :project` in the samples model.

@happyimadesignr in projects view you have "uploaded x time ago". At the time we went we creation date. But might make sense to be date last time a sample was added/removed from that project? Or event user changes?

Reply:
@tfrcarvalho my understanding is that all types of visualizations are immutable with respect to samples. That is, they only use the samples specified at creation. 

My thinking for `updated_at` was to show when the view state of a vis has been changed.

Reply:
This changes `ProjectsView` reference date, which is unrelated with visualizations...

Reply:
Sorry, this change was totally accidental then! But I think some your points still stand about whether to order projects by created or updated.